lib/src/main/java/com/auth0/spring/security/api/JwtAuthenticationEntryPoint.java

```diff
@@ -9,8 +10,24 @@
 import java.io.IOException;
 
 public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
+
+    final String audience;
+    final String issuer;
+
+    public JwtAuthenticationEntryPoint(String audience, String issuer) {
+        this.audience = audience;
+        this.issuer = issuer;
+    }
+
     @Override
     public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException, ServletException {
+        String modifiedIssuer = this.issuer.endsWith("/") ? this.issuer.substring(0, this.issuer.length() - 1) : this.issuer;
```

Comment:
Is this to ensure that the header value won't have double "/"? We will need to account for potential null values (related to the above comment regarding needing a default constructor, in which `audience` and `issuer` would be null). Also would be good to make sure all cases are covered by tests, that might be why the code coverage check is failing.